pkg/smartcontract/nef/nef_test.go

```diff
@@ -29,6 +29,13 @@ func TestEncodeDecodeBinary(t *testing.T) {
 		checkDecodeError(t, expected)
 	})
 
+	t.Run("invalid reserved", func(t *testing.T) {
+		bytes, err := testserdes.EncodeBinary(expected)
+		require.NoError(t, err)
+		bytes[4+32+32] = 1 // set the first reserved byte to 1
+		require.Error(t, testserdes.DecodeBinary(bytes, &File{}))
```

Comment:
I don't like arbitrary offsets. Maybe smth like this?
```
invalid := *expected
invalid.Reserved = 1
checkDecodeError()
```

Reply:
Not sure, because we actually don't have `Reserved` field. It is just four bytes which are being added during encoding. I can add `Reserved` field to the `File` structure, but it will require additional safety check during encoding.